param.c

```diff
@@ -354,6 +357,11 @@ void read_parameter_file(char *fname)
         All.CP.Omega0 = param_get_double(ps, "Omega0");
         All.CP.OmegaBaryon = param_get_double(ps, "OmegaBaryon");
         All.CP.OmegaLambda = param_get_double(ps, "OmegaLambda");
+        All.CP.Omega_fld = param_get_double(ps, "Omega_fld");
+        if(All.CP.OmegaLambda > 0 && All.CP.Omega_fld > 0)
```

Comment:
We shall look into sharing the cosmology construction code in GenIC and the main code..

Reply:
One thing we could do is to specify the cosmology in GenIC, store it in the header, and then just read it in the main code.

Reply:
It just creates another parser -- from bigfile attr to cosmology..